frontend/src/metabase/nav/components/search/RecentsList/RecentsList.tsx

```diff
@@ -67,6 +69,33 @@ export const RecentsList = ({ onClick, className }: RecentsListProps) => {
         results={wrappedResults}
         onClick={onContainerClick}
       />
+      <Flex
+        px="1rem"
+        py=".5rem"
+        gap=".5rem"
+        align="center"
+        bg="#f9fbfc"
+        style={{
+          borderBottomLeftRadius: ".5rem",
+          borderBottomRightRadius: ".5rem",
+          borderTop: "1px solid #f0f0f0",
+        }}
+      >
+        <Text
+          p="0.25rem"
+          bg={color("bg-light")}
+          fw={700}
+          fz="8pt"
+          lh="8pt"
+          style={{
+            borderRadius: "0.25rem",
+            border: `1px solid ${color("border")}`,
+          }}
+        >{`${metaKey} + K `}</Text>
+        <Text size="sm" c={color("text-light")} fw={700} tt="uppercase">
+          Open command palette
+        </Text>
+      </Flex>
```

Comment:
maybe this could be its own component? all the inline styling makes this a little cluttered. could also encapsulate the shortcut key display logic there

Reply:
does this need to be hidden when embedded?

Reply:
Good point on the embedding. And sure, I'll move it into it's own component